src/scripts/chimera-cam

```diff
@@ -598,6 +607,21 @@ class ChimeraCam (ChimeraCLI):
             self.out(40 * "=")
 
 
+    def _downloadImage(self,image):
```

Comment:
The idea is nice and I like it, but I think it is better to implement this directly on Image class, it might be useful somewhere else where you want to make a remote image local.
 ...